<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_furniture</class>
 <widget class="QDialog" name="Dialog_furniture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление фурнитуры</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(209, 238, 252);</string>
  </property>
  <widget class="QLabel" name="label_select_furniture">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите фурнитуру</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_furniture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 186, 191);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_furniture">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 186, 191);</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 186, 191);</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_new_furniture">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 186, 191);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
